react/src/pages/datasets/DatasetInfoDialog.tsx

```diff
@@ -0,0 +1,235 @@
+import React, { useEffect, useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Dialog, DialogTitle, DialogContent, Paper, Typography,Grid, IconButton, Collapse, Button, List, ListItem, ListItemText, ListItemIcon, Box } from '@material-ui/core';
+import { Close, ShowChart, ExpandLess, ExpandMore } from '@material-ui/icons';
+import { Dataset, Analysis, Sample } from '../utils';
+
+const useStyles = makeStyles(theme => ({
+    datasetInfo: {
+        paddingLeft: theme.spacing(2),
+        paddingRight: theme.spacing(2),
+    },
+    paper: {
+        padding: theme.spacing(2),
+        marginBottom: theme.spacing(3),
+    },
+    listPaper: {
+        padding: theme.spacing(1),
+        margin: theme.spacing(1)
+    },
+    root: {
+        margin: 0,
+        padding: theme.spacing(2),
+    },
+    closeButton: {
+        position: 'absolute',
+        right: theme.spacing(1),
+        top: theme.spacing(1),
+        color: theme.palette.grey[500],
+    },
+    grid: {
+        paddingBottom: theme.spacing(2)
+    },
+    box: {
+        padding: theme.spacing(2),
+        margin: theme.spacing(1)
+    }
+}));
+
+interface DialogTitleProps {
+    id: string;
+    children: React.ReactNode;
+    onClose: () => void;
+}
+
+const DialogHeader = ((props: DialogTitleProps) => {
+    const { children, onClose, ...other } = props;
+    const classes = useStyles();
+    return (
+        <DialogTitle disableTypography className={classes.root} {...other}>
+            <Typography variant="h6">{children}</Typography>
+            {onClose ? (
+                <IconButton aria-label="close" className={classes.closeButton} onClick={onClose}>
+                    <Close />
+                </IconButton>
+            ) : null}
+        </DialogTitle>
+    );
+});
+
+const getInfo = (subtitle: string, value: string | number | null) => {
```

Comment:
This function is also used in ParticipantDetailDialog component in the dashboard page, so you could move it to utils.tsx